go/worker/client/service.go

```diff
@@ -238,6 +239,31 @@ func (s *service) GetTransactionsWithResults(ctx context.Context, request *api.G
 	return results, nil
 }
 
+// Implements api.RuntimeClient.
+func (s *service) GetUnconfirmedTransactions(_ context.Context, runtimeID common.Namespace) ([][]byte, error) {
+	rt := s.w.commonWorker.GetRuntime(runtimeID)
+	if rt == nil {
+		return nil, api.ErrNotFound
+	}
+
+	// Get currently pending transactions from the runtime's transaction pool.
+	pendingTxs := rt.TxPool.GetTxs()
+
+	// Sort transactions by first seen time (ascending).
+	sort.SliceStable(pendingTxs, func(i, j int) bool {
+		a := pendingTxs[i]
+		b := pendingTxs[j]
+		return a.FirstSeen().Before(b.FirstSeen())
+	})
```

Comment:
I wouldn't change the sort order as these should be sorted by priority already.

Reply:
Oh, OK, I'll remove this.